litellm/main.py

```diff
@@ -117,7 +117,31 @@ def create(self, messages, model=None, **kwargs):
     return response  
 
 @client
-async def acompletion(*args, **kwargs):
+async def acompletion(
```

Comment:
can we have a list, `litellm.completion_params` that way we can re-use this. Or even if there was a way to access the `completion` args here. 

Context: Litellm.completion() gets updated frequently, and it would be better if we don't constantly need to make the same update to completion and acompletion every time we add a new param